packages/@webex/internal-plugin-voicea/src/voicea.types.ts

```diff
@@ -78,7 +78,7 @@ interface IVoiceaChannel {
   setSpokenLanguage: (languageCode: string) => Promise<void>;
   requestLanguage: (languageCode: string) => void;
   turnOnCaptions: () => undefined | Promise<void>;
-  toggleTranscribing: (activate: boolean) => undefined | Promise<void>;
+  toggleTranscribing: (activate: boolean, spokenLanguage: string) => undefined | Promise<void>;
```

Comment:
as mentioned in the other file, spokenLanguage could be an enum? This would help prevent us from sending invalid data

Reply:
We cannot make the languageCodes as enum are they may vary from time to time and it comes from the backend. We don't define it in the SDK